pillar.example

```diff
@@ -72,7 +76,7 @@ hana:
         user: 'SYSTEM'
         password: 'Qwerty1234'
 
-    - host: hana02
+    - host: hana03
```

Comment:
Don't change this. This is to show that you can have more than 1 SAP instance in the same host (this is the cost optimized scenario in fact)